src/foundation.cs

```diff
@@ -10980,6 +10980,10 @@ partial interface NSFileManager {
 #else
 		bool GetRelationship (out NSURLRelationship outRelationship, NSUrl directoryURL, NSUrl otherURL, out NSError error);
 #endif
+
+		[NoWatch][NoTV][NoiOS][Mac (10, 11)]
+		[Export ("unmountVolumeAtURL:options:completionHandler:")]
+		void UnmountVolume (NSUrl url, NSFileManagerUnmountOptions mask, Action<NSError> completionHandler);
```

Comment:
_because I like my code public and my mistakes private_ -> https://github.com/spouliot/Touch.Unit ;-)